### PR TITLE
chore(release): 3.13.0

### PR DESCRIPTION
**Pull request by Laura-Danielle**
## overview
PR to kick off the release of 3.13.0

- [ ] Necessary buildroot PRs merged
- [ ] Alpha RC tagged and released
- [ ] QC Completed

## Links

https://coda.io/d/Opentrons-App-API-Release-QA-Worksheet_dHwyPU5HADk/3-13-x_su4hu#_luoPM

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
 ## Bug Fixes
 
-- We fixed an issue with the tip probe algorithm to correct for some middle-switch misses
-- We removed some badly behaving code that was arbitrarily causing "large" (>4.5 MB) protocol uploads and simulations to fail
-- We fixed some problems with our Jupyter notebook configuration that were broken during our operating system update in 3.11.x
-  - File deletion has been fixed
-  - Migration of pre-3.11 Jupyter Notebooks to the new location `/var/lib/jupyter/notebooks` has also been fixed
-- Removed an incorrect Network Manager setting that was blocking auto-IP detection for _all_ Ethernet interfaces instead of only the built-in one
-  - Plugging in external USB-to-Ethernet adapters to connect your OT-2 to a wired network should work again
+- Some typos in our documentation were fixed
```
Comment (sfoster1) [inline api/release-notes.md]: Maybe add `- Fixed issues that could result in the MAC addresses of the robot's network interfaces changing over time. Disconnect from and reconnect to your current wireless network to get this change. Note that on this first disconnect and reconnect, the MAC address may change.`

Comment (sfoster1) [reply api/release-notes.md]: - Fixed an issue that could result in errors during attach pipette, which would manifest as unexpected homes.

File: api/release-notes.md [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
+- Problems with pipette height following deck calibration have been fixed
```
Comment (sfoster1) [inline api/release-notes.md]:
```suggestion
- Problems with pipette height following deck calibration have been fixed. If you ran the command-line variant of deck calibration in 3.12.0, you should rerun it after installing this update.
```

File: app-shell/build/release-notes.md [hunk 3]
```diff
@@ -1,38 +1,23 @@
...
+# Opentrons App Changes from 3.12.0 to 3.13.0
 
-**Note for macOS users**: This update is not compatible with macOS 10.9 Mavericks. If you are using 10.9, please upgrade to a later version of macOS (available for free from Apple) to continue to receive Opentrons updates.
+**Note for macOS users**: The Opentrons App is no longer compatible with macOS 10.9 Mavericks. If you are using 10.9, please upgrade to a later version of macOS (available for free from Apple) to continue to receive Opentrons updates.
 
 For more details about this release, please see the full [technical change log][changelog]
 
 [changelog]: https://github.com/Opentrons/opentrons/blob/edge/CHANGELOG.md
 
 ## New Features
```
Comment (sfoster1) [inline app-shell/build/release-notes.md]: We don't have a lot of changes on the app side (or at least not visible ones) this release, but we do have

- the new attach pipette menu refresh
- disable run start button with missing modules

@Kadee80 @mcous anything I'm missing? How should we format this? Keep a section empty?

File: api/release-notes.md [hunk 4]
```diff
@@ -1,25 +1,20 @@
...
 ## Bug Fixes
 
-- We fixed an issue with the tip probe algorithm to correct for some middle-switch misses
-- We removed some badly behaving code that was arbitrarily causing "large" (>4.5 MB) protocol uploads and simulations to fail
-- We fixed some problems with our Jupyter notebook configuration that were broken during our operating system update in 3.11.x
-  - File deletion has been fixed
-  - Migration of pre-3.11 Jupyter Notebooks to the new location `/var/lib/jupyter/notebooks` has also been fixed
-- Removed an incorrect Network Manager setting that was blocking auto-IP detection for _all_ Ethernet interfaces instead of only the built-in one
-  - Plugging in external USB-to-Ethernet adapters to connect your OT-2 to a wired network should work again
+- Some typos in our documentation were fixed.
```
Comment (Laura-Danielle) [inline api/release-notes.md]: Do we want to note that we've modified our log downloading process to prevent crashes?

Comment (sfoster1) [reply api/release-notes.md]: nah don't think we care

File: app-shell/build/release-notes.md [hunk 5]
```diff
@@ -1,38 +1,24 @@
-# Opentrons App Changes from 3.11.4 to 3.12.0
+# Opentrons App Changes from 3.12.0 to 3.13.0
 
-**Note for macOS users**: This update is not compatible with macOS 10.9 Mavericks. If you are using 10.9, please upgrade to a later version of macOS (available for free from Apple) to continue to receive Opentrons updates.
+**Note for macOS users**: The Opentrons App is no longer compatible with macOS 10.9 Mavericks. If you are using 10.9, please upgrade to a later version of macOS (available for free from Apple) to continue to receive Opentrons updates.
 
 For more details about this release, please see the full [technical change log][changelog]
```
Comment (Laura-Danielle) [inline app-shell/build/release-notes.md]: The new download flow should be noted. A modal no longer pops up for you to save your logs, but instead a progress spinner appears and logs go directly into your downloads folder with their default names of `serial` and `api` logs

**Pull request merged**